content/concept-keys.md

```diff
@@ -50,7 +50,8 @@ You can use this generated key pair in one of three ways: as a [master key pair]
 
 **Key Type**
 
-The field `key_type` indicates what [cryptographic signing algorithm](#signing-algorithms) was used to generate this key pair. You can specify the `key_type` in the request to
```

Comment:
@mDuo13 - Made a small tweak to finish this. Did I get where you were going?

Reply:
Yep. Thanks.